src/components/ConfirmationModal.vue

```diff
@@ -67,11 +67,31 @@ export default {
 
 <style lang="scss" scoped>
 .confirm-modal {
-	padding: 20px;
+	padding: 24px 32px;
+
+	h2 {
+		margin-bottom: 20px;
+		font-size: 24px;
+		font-weight: 600;
+	}
 
 	&__buttons {
 		display: flex;
-		justify-content: space-between;
+		justify-content: flex-end;
+		gap: 16px;
+		margin-top: 30px;
+		padding-top: 20px;
+
+		:deep(button) {
```

Comment:
also, you dont need the v:deep. You only need to add space on top of the buttons, and you need to make sure the buttons have space in between.